docs/examples/Parameters/Parameters.ipynb

```diff
@@ -13,23 +15,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import qcodes as qc\n",
-    "import numpy as np"
+    "import numpy as np\n",
+    "from qcodes.instrument.parameter import Parameter, ArrayParameter, MultiParameter, ManualParameter\n",
+    "from qcodes.utils import validators"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "QCoDeS provides 3 classes of parameter built in:\n",
+    "QCoDeS provides the following classes of built-in parameters:\n",
     "- `Parameter` represents a single value at a time\n",
     "  - Example: voltage\n",
-    "- `ArrayParameter` represents an array of values of all the same type that are returned all at once\n",
+    "- `ParameterWithSetpoints` is intended for array-values parameters.\n",
+    "    This Parameter class is intended for anything where a call to the instrument\n",
+    "    returns an array of values. [This notebook](Simple-Example-of-ParameterWithSetpoints.ipynb)\n",
+    "    gives more detailed examples of how this parameter can be used.\n",
+    "- `ArrayParameter` represents an array of values of all the same type that are returned all at once. \n",
     "  - Example: voltage vs time waveform\n",
+    "  - **NOTE:** This is an older base class for array-valued parameters. For any new driver we strongly recommend using `ParameterWithSetpoints` class which is both more flexible and significantly easier to use. Refer to notebook on [writing drivers with ParameterWithSetpoints](http://qcodes.github.io/Qcodes/examples/Parameters/Simple-Example-of-ParameterWithSetpoints.html)   \n",
     "- `MultiParameter` represents a collection of values with different meaning and possibly different dimension\n",
     "  - Example: I and Q, or I vs time and Q vs time\n",
     "\n",
-    "which are described in the \"Creating Instrument Drivers\" tutorial."
+    "Parameters are described in detail in the [Creating Instrument Drivers](http://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Instrument-Drivers.html) tutorial."
```

Comment:
Please use direct links to other notebooks as documented here https://nbsphinx.readthedocs.io/en/0.4.3/markdown-cells.html#Links-to-Other-Notebooks